Expensify/App
# Workspace pages copy changes and growl on save

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/180784
$ https://github.com/Expensify/Expensify/issues/180786
$ https://github.com/Expensify/Expensify/issues/180789
$ https://github.com/Expensify/Expensify/issues/180793
$ https://github.com/Expensify/Expensify/issues/180800
$ https://github.com/Expensify/Expensify/issues/180802
$ https://github.com/Expensify/Expensify/issues/180804

### Tests / QA Steps
1. Create a new workspace if you don't already have one
1. Go to `Settings` -> `Your workspace`
1. Verify that you see `General settings` and a menu item.
1. Click on `General settings`
1. Verify that the title on the settings page is `General settings`, not `Edit workspace`
1. Verify that the `Default currency` drop-down has a lowercase `C` in currency
1. Change the default currency, then hit save.
1. Verify that a Growl message appears which says `Your workspace settings were successfully chagned!`
1. Go to `Settings` -> `Your workspace` -> `Reimburse receipts`
1. Verify that there is a section title reading `Unlock next-day reimbursements` instead of `Unlock next day reimbursements`
1. Go to `Settings` -> `Your workspace` -> `Send invoices`
1. Verify that there is a button reading `Send invoice`, rather than `Send invoices`
1. Verify that the section title says `Unlock online invoice collection`, rather than `Unlock online invoices collection`.
1. Click on `Settings` -> `Your workspace` -> `Manage members` and verify that the page title says `Manage members`, not `Members`.
1. Go to `Settings` -> `Preferences` and change your preferred language to Español.
1. Go to `Configuración` -> `Your workspace` -> `Configuración general`
1. Verify that the title on the page is `Configuración general`
1. Change the default currency, then hit save.
1. Verify that a Growl message appears which says `¡Tus configuraciónes del espacio de trabajo se han guardado correctamente!`.

### Tested On


### Screenshots


#### Web
|Testing step(s)|Screenshot|
|---|---|
|5-6|<img src="https://user-images.githubusercontent.com/47436092/136852891-ef8a0a8b-b41b-4313-bfd8-c1f18c1959e3.png" alt="" width="350px" />|
|8|<img src="https://user-images.githubusercontent.com/47436092/136852993-5565208f-3a2f-453d-b170-251c7449a37d.png" alt="" width="350px" />|
|10|<img src="https://user-images.githubusercontent.com/47436092/136853097-5edba823-5800-4f72-a72c-b73e1ddd8577.png" alt="" width="350px" />|
|12-13|<img src="https://user-images.githubusercontent.com/47436092/136853174-49468e60-fd4a-4996-ac18-f52cce2ddeae.png" alt="" width="350px" />|
|14|<img src="https://user-images.githubusercontent.com/47436092/136853207-fa655c2c-6b08-4359-a333-9a8356fd6ad2.png" alt="" width="350px" />|
|17|<img src="https://user-images.githubusercontent.com/47436092/136853381-855147f8-524b-42d6-a279-c18bb2d561a0.png" alt="" width="350px" />|
|19|<img src="https://user-images.githubusercontent.com/47436092/136853502-09d2de60-a453-471c-b508-bbc5508ddac2.png" alt="" width="350px" />|

#### Mobile Web


#### Desktop


#### iOS


#### Android
